test-case/check-playback.sh

```diff
@@ -88,6 +92,10 @@ do
             do
                 dlogi "Testing: (Round: $round/$round_cnt) (PCM: $pcm [$dev]<$type>) (Loop: $i/$loop_cnt)"
                 dlogc "aplay -D$dev -r $rate -c $channel -f $fmt_elem -d $duration $file -v -q"
+		#sleep random 0-7 seconds before aplay to cover triggering aplay from both runtime PM suspend and active state.
```

Comment:
improve the comment here, may change to `sleep 0~random_range seconds before running aplay/arecord, then aplay/arecord can be run under dsp active or suspended status randomly`